src/main/resources/events/common.json

```diff
@@ -10,5 +10,15 @@
     {
         "name": "open_to_lan",
         "type": "open_to_lan"
```

Comment:
Should probably have `"repeatable" :false` on all of the events for completeness but meh